contracts/extensions/votingReputation/IVotingReputation.sol

```diff
@@ -281,10 +281,14 @@ interface IVotingReputation is IColonyExtension, VotingReputationDataTypes {
   ) external view returns (uint256 _rewardMin, uint256 _rewardMax);
 
   /// @notice Return a summary of the multicall action
+  /// @param _colonyNetwork The address of the colony network
+  /// @param _colony The address of the colony this extension is installed in
   /// @param _action The id of the motion
   /// @param _altTarget The address of the altTarget, or 0x0 if none exists
   /// @return _summary A summary of the multicall
   function getActionSummary(
```

Comment:
I feel like we actually shouldn't change this interface - inviting ourselves to trust user input about what `colonyNetwork` is seems like it will set us up for disaster down the line.

Reply:
I think that's a good point. Are `colony` and `colonyNetwork` not stored on this extension already?

Reply:
Re-reading the changes, I don't think this is a risk -- neither value is user-supplied.

Reply:
I agree for now it's fine, but it's down-the-line I'm worrying about, where we see this interface and use it without thinking. I'll just make a public wrapper for the mixin, which will still take the parameters.